scripts/gcp_setup/environment/gcloud_env.py

```diff
@@ -146,6 +146,9 @@ def __init__(self, **kwargs):
         self.notification_recipient_email = (
             kwargs.get('notification_recipient_email'))
         self.gsuite_superadmin_email = kwargs.get('gsuite_superadmin_email')
+        self.host_project_id = kwargs.get('host_project', self.project_id)
+        self.vpc_name = kwargs.get('vpc') or 'default'
+        self.subnetwork = kwargs.get('subnet') or 'default'
```

Comment:
ditto as above:
```
self.subnetwork_name = kwargs.get('subnet_name', 'default')
```

Reply:
Same as above. I'll make the change if that's the direction the rest of the file is supposed to go.